app/views/questionnaire.py

```diff
@@ -67,17 +67,14 @@ def before_questionnaire_request():
     logger.bind(tx_id=metadata['tx_id'])
 
     values = request.view_args
+
+    if check_multiple_survey(metadata, values):
```

Comment:
As discussed, this can be removed.

Reply:
As we've now found we need this for page refresh.